packages/data-transport-layer/src/services/l1-ingestion/handlers/sequencer-batch-appended.ts

```diff
@@ -243,6 +243,31 @@ const maybeDecodeSequencerBatchTransaction = (
   let decoded = null
   let type = null
 
+  try {
+    // Try to decode as RLP first. This function will throw if the transaction can't be properly
+    // decoded as RLP and we'll get bumped down to the next set of possible decodings.
+    const decodedTx = ethers.utils.parseTransaction(transaction)
+
+    return {
+      type: 'EIP155',
+      decoded: {
+        nonce: BigNumber.from(decodedTx.nonce).toNumber(),
+        gasPrice: BigNumber.from(decodedTx.gasPrice).toNumber(),
+        gasLimit: BigNumber.from(decodedTx.gasLimit).toNumber(),
+        target: toHexString(decodedTx.to), // Maybe null this out for creations?
+        data: toHexString(decodedTx.data),
+        sig: {
+          v: BigNumber.from(decodedTx.v).toNumber(),
+          r: toHexString(decodedTx.r),
+          s: toHexString(decodedTx.s),
+        },
+        type: 0, // EIP155 legacy holdover.
+      },
+    }
+  } catch (err) {
+    // Do nothing, fall back to legacy decode.
+  }
+
   try {
```

Comment:
We don't need this decoding anymore because we don't have to be backwards compatible

Reply:
Assuming this is added to semver version `0.3.x`